### DISCUSSION TITLE
Microsoft_SQL_Server Unknown metric windowssqlinstances.discovery

### DISCUSSION THREAD

**Discussion by olistu** [zabbix/community-templates#282] [Show and tell]
Hi,

I want to use the Template "App MS SQL Instance Discovery" on my Zabbix installation. Zabbix server run on RedHat 9. The MS-SQL Server is Windows Server 2019 with MS SQL Server 2019 and 2016.

But the Discovery rule give the following error: "Unknown metric windowssqlinstances.discovery". What is wrong ? Or what I forgot ?

I need some help. Thanks
Regards
Olivier

**Comment by CanutaOS** [comment]
Hi, 

Didi you manage to find a solution for this? 

Thank you

**Reply by olistu** [reply]
> Hi,
> 
> Didi you manage to find a solution for this?
> 
> Thank you

Hi,

No finally, I use the MSSQL templates. See page https://www.zabbix.com/integrations/mssql

**Reply by CanutaOS** [reply]
Thank you for your reply. Did you manage to make it work for instances that listen on dynamic ports?
